### PR TITLE
Release quinteros-2

### PR DESCRIPTION
**Pull request by Fryguy**
@bdunne Please review.  Once you are ready to go, this has the details.

### CODE REVIEW

**Comment by Fryguy** [comment]
@bdunne If there are special upgrade steps can you list them here so I can add them to the blog post?

**Comment by bdunne** [comment]
I think we just need this:
```
sed -i 's/mirrorlist=/#mirrorlist=/g' /etc/yum.repos.d/CentOS-*.repo
sed -i 's/#baseurl=http:\/\/mirror/baseurl=http:\/\/vault/g' /etc/yum.repos.d/CentOS-*.repo
```

**Comment by Fryguy** [comment]
Ok, so if they are doing an RPM upgrade then before hand just need those 2 steps?

**Comment by bdunne** [comment]
Yes

**Comment by Fryguy** [comment]
@bdunne Updated.

File: site/_posts/2024-11-14-quinteros-2-cve-2024-43191-cve-2023-46715.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+### Medium severity
+
+- [CVE-2023-46175 - Credentials logged in plaintext for some providers](https://github.com/ManageIQ/manageiq/security/advisories/GHSA-pfh3-vj6p-89h9).
+
+  Thanks to [@sigbjornaib](https://github.com/sigbjornaib) for finding and reporting this issue.
+
+## Upgrade Notes
+
+Due to the vaulting of CentOS Stream 8<sup>[[1]][[2]]</sup>, the existing RPM repo files are pointing to a location that no longer exists. As such,
+when doing an RPM upgrade to quinteros-2, there are a manual steps that need to be done first. Run the following 2 commands before upgrading, which will
```
Comment (bdunne) [inline site/_posts/2024-11-14-quinteros-2-cve-2024-43191-cve-2023-46715.md:29]:
```suggestion
when doing an RPM upgrade to quinteros-2, there are some manual steps that need to be done first. Run the following 2 commands before upgrading, which will
```

File: site/_posts/2024-11-14-quinteros-2-cve-2024-43191-cve-2023-46715.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+### Medium severity
+
+- [CVE-2023-46175 - Credentials logged in plaintext for some providers](https://github.com/ManageIQ/manageiq/security/advisories/GHSA-pfh3-vj6p-89h9).
+
+  Thanks to [@sigbjornaib](https://github.com/sigbjornaib) for finding and reporting this issue.
+
+## Upgrade Notes
+
+Due to the vaulting of CentOS Stream 8<sup>[[1]][[2]]</sup>, the existing RPM repo files are pointing to a location that no longer exists. As such,
```
Comment (bdunne) [inline site/_posts/2024-11-14-quinteros-2-cve-2024-43191-cve-2023-46715.md:28]:
```suggestion
Due to the vaulting of CentOS Stream 8<sup>[[1]][[2]]</sup>, the existing RPM repo files are pointing to a mirrorlist that no longer exists. As such,
```

File: site/_posts/2024-11-14-quinteros-2-cve-2024-43191-cve-2023-46715.md (hunk 1 above)
Comment (Fryguy) [reply site/_posts/2024-11-14-quinteros-2-cve-2024-43191-cve-2023-46715.md]: Updated

**Pull request merged**